pkgs/applications/video/kodi/addons/invidious/default.nix

```diff
@@ -6,7 +6,7 @@ buildKodiAddon rec {
   version = "0.1.0+matrix.1";
 
   src = fetchzip {
-    url = "https://mirrors.kodi.tv/addons/matrix/${namespace}/${namespace}-${version}.zip";
+    url = "https://mirrors.kodi.tv/addons/nexus/${namespace}/${namespace}-${version}.zip";
```

Comment:
While not a packaging issue, this addon has several problems:
1. the default invidious URL is no longer active (has been fixed upstream: https://github.com/TheAssassin/kodi-invidious-plugin/commit/5474b454d20f16e924a936a49e1c7735f5d9bacb )
2. the search does not work, so one can only see Trending and Popular videos

All in all, I don't know if this module is worth keeping